src/docs/release/breaking-changes/text-field-material-localizations.md

```diff
@@ -0,0 +1,122 @@
+---
+title: TextField requires Material Localizations
+description: TextFields will now assert for Material Localizations.
+---
+
+## Summary
+
+Instances of `TextField` must have a `MaterialLocalizations` present.
+Trying to instantiate  a `TextField` without the proper localizations will
+result in an assertion such as the following:
+
+```
+No MaterialLocalizations found.
+TextField widgets require MaterialLocalizations to be provided by a Localizations widget ancestor.
+Localizations are used to generate many different messages, labels, and abbreviations which are used
+by the material library.
+To introduce a MaterialLocalizations, either use a MaterialApp at the root of your application to
+include them automatically, or add a Localization widget with a MaterialLocalizations delegate.
+The specific widget that could not find a MaterialLocalizations ancestor was:
+  TextField
+```
+
+## Context
+
+If your `TextField` is within the scope of a `MaterialApp`, the
+`DefaultMaterialLocalizations` is already be instantiated and may suit your needs without having
+to make a change to your existing code.
+
+If your `TextField` is not within a `MaterialApp`, you may provide the localizations of
+your choosing using the `Localizations` widget.
+
+## Migration guide
+
+If you are seeing an assertion error, make sure locale information is being
+provided to your `TextField` through an ancestor `MaterialApp`, which will automatically provide
```

Comment:
If you are seeing an assertion error, make sure locale information is being provided to your `TextField` through an ancestor `MaterialApp`, which will automatically provide `MaterialLocalizations`, or with your own `Localizations` widget. =>
If you see an assertion error, make sure that locale information is available to the `TextField`, either through an ancestor `MaterialApp` (that automatically provides localizations), or by creating your own `Localizations` widget.

Though I'm a bit confused. This sentence here looks like there are THREE ways to do it, but above it seemed like only two...

Reply:
What is the third option you are interpreting?